samples/ef-core/json/json-columns.cs

```diff
@@ -0,0 +1,171 @@
+//This sample shows how to use EF Core JSON columns with ODP.NET and Oracle Database.
+//It creates an owned entity, inserts, queries, updates, and deletes JSON column data.
+//It requires Oracle EF Core 8 or higher. Oracle Database 21c and higher supports JSON columns.
+//Earlier database versions map aggregate types to NCLOB columns instead of JSON columns.
+
+//Specify the user, password, and data source in the connection string below.
+
+using Microsoft.EntityFrameworkCore;
+
+namespace ODPJsonColumns
+{
+    class Program
+    {
+        public class ContactDetails
+        {
+            public Address Address { get; set; } = null!;
+            public string? Phone { get; set; }
+        }
+    
+        public class Address
+        {
+            public Address(string street, string city, string postcode, string country)
+            {
+                Street = street;
+                City = city;
+                Postcode = postcode;
+                Country = country;
+            }
+      
+            public string Street { get; set; }
+            public string City { get; set; }
+            public string Postcode { get; set; }
+            public string Country { get; set; }
+        }
+    
+        public class Author
+        {
+            public int Id { get; set; }
+            public string? Name { get; set; }
+            public ContactDetails? Contact { get; set; }
+        }
+
+        public class AuthorContext : DbContext
+        {
+            public DbSet<Author> Authors { get; set; }
+
+            //To use Oracle database JSON columns, connect to Oracle Database 21c or higher version and
+            // specify OracleSQLCompatibility to DatabaseVersion21 or higher.
+            protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+            {
+                optionsBuilder.UseOracle("User Id=<USER>; Password=<PASSWORD>; Data Source=<DATA SOURCE>"
+                    , b => b.UseOracleSQLCompatibility(OracleSQLCompatibility.DatabaseVersion21)); ;
```

Comment:
doubled ;

Reply:
Good catch! Filed a PR to fix this issue: https://github.com/oracle/dotnet-db-samples/pull/343